cmd/ocm-backplane/login/login.go

```diff
@@ -183,6 +185,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
+	backplaneResolution, err := getBackplaneCNAME(bpURL)
+	if err != nil {
+		logger.Warn(err.Error())
+	} else {
+		logger.Debugf("Backplane URL resolves to %s \n", backplaneResolution)
+		logger.Debugf("Please refer to your design documentation to figure out the associated backplane server")
```

Comment:
nit - Let's make the message more generic

```suggestion
		logger.Debugf("To know the associated Backplane instance please refer to the Backplane Source wiki")
```